tools/codegen/dest/lazy_ir.py

```diff
@@ -273,11 +301,11 @@ def this_shape(i: int) -> str:
 
         return [f"""\
     {sig.decl(name=f"{self.class_method_name}::{metadata.kernel}")} {{
+        TORCH_LAZY_TIMED("LazyNativeFunctions::full_codegen");
```

Comment:
see my latest refactor of codegen (on master) where more stuff is 'customizable' being string variables in the GenLazyNativeFunc class.  I don't think XLA wants to customize this as they may not even care about it, but cc @JackCaoG in case he does. We probably want to make this a string variable too